src/littlefs/lfs.pyx

```diff
@@ -186,13 +201,58 @@ def removeattr(LFSFilesystem fs, path, type):
     raise NotImplementedError
 
 
+def _parse_mode(mode) -> LFSFileFlag:
```

Comment:
Would it be possible to consolidate the parsing code which is used here with the code used in LittleFS.open() ?
At least the string -> boolean flags conversion seems (should) be the same.